### DISCUSSION TITLE
About SDK was imported into Unity, and an error was reported

### DISCUSSION THREAD

**Discussion by Lihao7788** [sofa-framework/sofa#4289] [Build / Config / Environment]
Hello, I imported SofaUnityPackage into Unity2021.3.19, but there was no effect in the running scene, and the Console panel reported an error! Error: ExecutionEngineException: String conversion error: Illegal byte sequence encounted in the input.

SofaUnityAPI.SofaContextAPI.DisplayMessages () (at Assets/SofaUnity/Plugins/SofaUnityAPI/SofaContextAPI.cs:429)

SofaUnity.SofaContext.DoCatchSofaMessages () (at Assets/SofaUnity/Scripts/SofaContext.cs:564)

SofaUnity.SofaContext.ReconnectSofaScene () (at Assets/SofaUnity/Scripts/SofaContext.cs:618)

SofaUnity.SofaContext.Init () (at Assets/SofaUnity/Scripts/SofaContext.cs:405)

SofaUnity.SofaContext.StartSofa () (at Assets/SofaUnity/Scripts/SofaContext.cs:312)

SofaUnity.SofaContext.Awake () (at Assets/SofaUnity/Scripts/SofaContext.cs:258)

Who can tell me what to do next

**Comment by epernod** [comment]
Hi,

I think this is a duplicated discussion to the #4287 no ?
If I'm wrong could you open a discussion into this repository: https://github.com/InfinyTech3D/SofaUnity-Renderer/discussions

Thank you

**Comment by Lihao7788** [comment]
Hello, I have created a question in the new forum, please check it




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "sofa-framework/sofa"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年11月14日(星期二) 晚上11:49
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [sofa-framework/sofa] About SDK was imported into Unity, and an error was reported (Discussion #4289)





 
Hi,
 
I think this is a duplicated discussion to the #4287 no ?
 If I'm wrong could you open a discussion into this repository: https://github.com/InfinyTech3D/SofaUnity-Renderer/discussions
 
Thank you
